src/components/application_manager/src/application_impl.cc

```diff
@@ -654,32 +653,31 @@ void ApplicationImpl::WakeUpStreaming(
   using namespace protocol_handler;
   SDL_LOG_AUTO_TRACE();
 
-  // See the comment in StopStreaming(). Also, please make sure that we acquire
-  // streaming_stop_lock_ then xxx_streaming_suspended_lock_ in this order!
   sync_primitives::AutoLock lock(streaming_stop_lock_);
 
   if (ServiceType::kMobileNav == service_type) {
-    {  // reduce the range of video_streaming_suspended_lock_
-      sync_primitives::AutoLock auto_lock(video_streaming_suspended_lock_);
-      if (video_streaming_suspended_) {
-        application_manager_.OnAppStreaming(app_id(), service_type, true);
-        application_manager_.ProcessOnDataStreamingNotification(
-            service_type, app_id(), true);
-        video_streaming_suspended_ = false;
-      }
+    if (video_streaming_stopped_) {
+      return;
+    }
+    if (video_streaming_suspended_) {
+      application_manager_.OnAppStreaming(app_id(), service_type, true);
```

Comment:
@ychernysheva please add a debug log the video streaming will be resumed after suspension

Reply:
Done in ad747fa